dotcom-rendering/src/components/SecureSignup.importable.tsx

```diff
@@ -343,41 +344,30 @@ export const SecureSignup = ({ newsletterId, successDescription }: Props) => {
 			>
 				<Label
 					text="Enter your email address"
-					cssOverrides={[
-						labelStyles,
-						css`
-							display: ${!signedInUserEmail
-								? 'inline-block'
-								: 'none'};
-						`,
-					]}
+					cssOverrides={[labelStyles]}
+					style={
+						signedInUserEmail
+							? { display: 'none' }
+							: { display: 'inline-block' }
+					}
```

Comment:
You can also set the value to `undefined` to let it just be the default value: 

```suggestion
					style={{
						display:
						signedInUserEmail ? 'none' : undefined
					}}
```